### DISCUSSION TITLE
Date literal?

### DISCUSSION THREAD

**Discussion by ModProg** [nushell/nushell#3926] [Q&A] [answered]
Is is possible to directly specify a date?

e.g. something like `2021-12-24 18:00`

**Comment by kubouch** [comment, marked as answer]
@ModProg You could do something like `echo "2020-12-11" | str to-datetime` and further modify it with `date format`.  There was a question about dates previously which you might find useful: https://github.com/nushell/nushell/discussions/3476 .

**Reply by ModProg** [reply]
I thought maybe there existed something with less typing, but maybe that's not useful often enough to justify it's own syntax.

**Reply by kubouch** [reply]
I think it might be possible to add a date/time literal, for example following the ISO standard. But like you said, it would be a lot of work for something that perhaps is not used that often.
